go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -65,6 +65,7 @@ const (
 
 	merchantKeyspace = "merchant-type"
 	maxWait          = 10 * time.Second
+	BypassLagCheck   = true // temporary fix for flakiness seen only in CI when lag check is introduced
```

Comment:
nit: this doesn't need to be exported, but if this is temporary, not a big deal.